javascript/builtins/Array.json

```diff
@@ -1884,7 +1884,7 @@
                   "version_added": "52"
                 },
                 "firefox_android": {
-                  "version_added": false
+                  "version_added": "56"
                 },
```

Comment:
This was in 56, while it appeared on desktop in 52. Is that intentional? Is there some special story here?